spring-cloud-gcp-data-datastore/src/main/java/org/springframework/cloud/gcp/data/datastore/core/DatastoreTemplate.java

```diff
@@ -140,32 +142,35 @@ public void setApplicationEventPublisher(ApplicationEventPublisher applicationEv
 
 	@Override
 	public <T> T save(T instance, Key... ancestors) {
-		return save(instance, new HashSet<>(), ancestors);
-	}
-
-	private <T> T save(T instance, Set<Key> persisted, Key... ancestors) {
-		saveAll(Collections.singletonList(instance), persisted, ancestors);
+		List<T> instances = Collections.singletonList(instance);
```

Comment:
this variable seems unnecessary

Reply:
It's used twice in the next line so I gave it its own name

Reply:
I see, makes sense